intellij/src/saros/intellij/editor/LocalEditorHandler.java

```diff
@@ -207,8 +224,9 @@ public void saveDocument(@NotNull SPath path) {
    * @param project the project in which to activate the editor
    * @param file the file whose editor was activated or <code>null</code> if there is no editor open
    */
-  public void activateEditor(@NotNull Project project, @Nullable VirtualFile file) {
-    if (file == null) {
+  public void activateEditor(
```

Comment:
Explanation of the fileEditor parameter ?

Reply:
Removed during adjustment of PR to correctly handle closing non-text editors and moving of non-text files.